src/transforms/apply-operation.js

```diff
@@ -32,7 +32,8 @@ const OPERATIONS = {
   set_node: setNode,
   split_node: splitNode,
   // Selection operations.
-  set_selection: setSelection
+  set_selection: setSelection,
+  set_document_data: setDocumentData
```

Comment:
Does the `set_node` operation now work here already for the document? I think it should, or we should make it work.